ipaserver/secrets/kem.py

```diff
@@ -85,8 +85,8 @@ def check_host_keys(self, host):
 
         ldap_filter = self.build_filter(IPA_CHECK_QUERY, {'host': host})
         r = conn.search_s(self.keysbase, scope, ldap_filter)
-        if len(r) != 1:
-            raise ValueError("Incorrect number of results (%d) searching for"
+        if len(r) < 1:
```

Comment:
Hello,

I think that `if not r:` is more pythonic and I'd also rephrase the error message, that "No public keys were found ...."

It is just nitpick, thanks.